apps/src/studio/studio.js

```diff
@@ -2495,7 +2497,7 @@ Studio.reset = function (first) {
   };
 
   // Reset the record of the last direction that the user moved the sprite.
-  Studio.lastMoveSingleDir = null;
```

Comment:
You might need it again, but it appears that in the current code this variable is no longer used.  

(It was used originally in https://github.com/code-dot-org/code-dot-org/commit/b7a89d649ca13db6b7f43b3640b758571e1d3b76#diff-eeb7666ffd9b0becfc57bba206a98594R4454)